LoopKitUI/Views/Settings Editors/BasalRateScheduleEditor.swift

```diff
@@ -50,6 +50,23 @@ public struct BasalRateScheduleEditor: View {
         self.save = save
         self.mode = mode
     }
+    
+    public init(
+        viewModel: TherapySettingsViewModel,
+        mode: PresentationMode = .acceptanceFlow
+    ) {
+        self.init(
+            schedule: viewModel.therapySettings.basalRateSchedule,
+            supportedBasalRates: viewModel.pumpSupportedIncrements!.basalRates ,
+            maximumBasalRate: viewModel.therapySettings.maximumBasalRatePerHour,
+            maximumScheduleEntryCount: viewModel.pumpSupportedIncrements!.maximumBasalScheduleEntryCount,
+            syncSchedule: viewModel.syncPumpSchedule,
+            onSave: { newRates in
+                viewModel.saveBasalRates(basalRates: newRates)
```

Comment:
The `.settings` TherapySettings screen will want to provide an `onSave` hook.  I recommend passing that in to the initializer. (applies to all of these initializers)